### PR TITLE
Convert to Gradle project with native component and add native-image support

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/3692

### CODE REVIEW

File: ballerina/Package.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ import ballerinax/mssql;
 import ballerinax/mssql.driver as _;
 ```
 
+## Report issues
+
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina standard library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+
 # Useful Links
 * Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
```
Comment (niveathika) [inline ballerina/Package.md:28]: Change to discord

Comment (kaneeldias) [reply ballerina/Package.md]: Fixed in [3a3bc95](https://github.com/ballerina-platform/module-ballerinax-mssql.driver/pull/12/commits/3a3bc95c2762137089607341f212e75e7d544345)

File: ballerina/Module.md [hunk 2]
```diff
@@ -18,7 +18,3 @@ import ballerina/sql;
 import ballerinax/mssql;
 import ballerinax/mssql.driver as _;
 ```
-
-# Useful Links
-* Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
```
Comment (niveathika) [inline ballerina/Module.md:23]: Why is this removed?

Comment (kaneeldias) [reply ballerina/Module.md:23]: We usually don't have the Useful links section in Module.md. Instead we have them in Package.md

e.g.
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Module.md
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Package.md

**Review by niveathika — APPROVED** [review]
